### PR TITLE
Fixes 2 accessiblity issues in PackageMetadataControl

### PR DESCRIPTION
**Pull request by dominoFire**
## Bug

Fixes: 
- https://github.com/NuGet/Home/issues/9077
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1049191
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/994497

Regression: No

Details: 
- Added `AutomationProperty.Name` to the hyperlinks in the control. The narrator can read more context about the link, not the hyperlink text itself.
- Removed extra tab stop in license container, by using `IsTabStop=False`

## Testing/Validation

Tests Added: No
Reason for not adding tests: Visual validation
Validation:  
- Accessibility Insights Run
- Narrator run

### CODE REVIEW

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml [hunk 1]
```diff
@@ -190,7 +190,7 @@ Background="{DynamicResource {x:Static nuget:Brushes.DetailPaneBackground}}"
         AutomationProperties.Name="{Binding Text}"
         Text="{x:Static nuget:Resources.Label_License}"/>
 
-      <ItemsControl Grid.Row="3" Grid.Column="1" ItemsSource="{Binding LicenseLinks}">
+      <ItemsControl Grid.Row="3" Grid.Column="1" ItemsSource="{Binding LicenseLinks}" IsTabStop="False">
```
Comment (donnie-msft) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml:193]: What does this do? Can the links still be accessed via tabbing?

**Review by donnie-msft — APPROVED** [review]
LGTM... had 1 question.

**Review by rrelyea — APPROVED** [review]
Since we are adding strings in this (and a few other recent checkins) we will need to work to make sure we get accessibility done on release build, then crank a new release build, and insert it before release.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml (hunk 1 above)
Comment (dominoFire) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml:193]: The link by itself forces a tab stop.  In this change, I'm removing a tab stop for the ItemsControl and the remaining tab stop would always be in the link.

**Pull request merged**